internal/manifests/collector/ingress.go

```diff
@@ -70,6 +70,38 @@ func Ingress(params manifests.Params) (*networkingv1.Ingress, error) {
 	}, nil
 }
 
+func ExtensionIngress(params manifests.Params) (*networkingv1.Ingress, error) {
+	name := naming.Ingress(params.OtelCol.Name)
+	labels := manifestutils.Labels(params.OtelCol.ObjectMeta, name, params.OtelCol.Spec.Image, ComponentOpenTelemetryCollector, params.Config.LabelsFilter())
+
+	ports, err := extensionServicePortsFromCfg(params.Log, params.OtelCol)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// if there are no ports, no ingress required
+	if len(ports) == 0 {
+		return nil, nil
+	}
+
+	rules := createSubdomainIngressRules(name, "", ports)
+
+	return &networkingv1.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        name,
+			Namespace:   params.OtelCol.Namespace,
+			Annotations: params.OtelCol.Spec.Ingress.Annotations, // can the spec annotations be used?
```

Comment:
What do you mean here?

Reply:
I was just wondering if we can use the `OtelCol.ObjectMeta.Annotations` instead of `Ingress.Annotations` as this function will be responsible for creating ingress for extension services.